develop/tutorials/articles/110-portlets/03-bean-portlet/00-bean-portlet-intro.markdown

```diff
@@ -1,63 +1,67 @@
+[all the new Portlet 3.0 features](https://portals.apache.org/pluto/v301/v3Features.html)
+in compliant portals, such as Liferay.
 
 Here you'll learn how to create and deploy a bean portlet. And if you're
 familiar with developing OSGi portlets, we'll help you relate what you already
-know to bean Portlet development---It's straightforward!
+know to bean portlet development---It's straightforward!
 
 ## Creating a bean portlet project [](id=creating-a-bean-portlet-project)
 
-CDI bean portlets have a Maven archetype.
+Bean portlets have a Maven archetype.
 
 - Group: `com.liferay`
 - Artifact: `com.liferay.project.templates.cdi.bean.portlet`
 - Version: `1.0.0`
 
 To create a bean portlet project, execute a Maven command like this one:  
 
-```mvn archetype:generate \
-  -DarchetypeGroupId=com.liferay \
-  -DarchetypeArtifactId=com.liferay.project.templates.cdi.bean.portlet \
-  -DarchetypeVersion=1.0.0 \
-  -DgroupId=com.mycompany \
-  -DartifactId=com.mycompany.demo.bean.portlet```
+    mvn archetype:generate \
+    -DarchetypeGroupId=com.liferay \
+    -DarchetypeArtifactId=com.liferay.project.templates.cdi.bean.portlet \
+    -DarchetypeVersion=1.0.0 \
+    -DgroupId=com.mycompany \
+    -DartifactId=com.mycompany.demo.bean.portlet
 
 Maven prompts you for a project author name, version, and portlet class
 name. For this example, the defaults are fine for everything else. 
 
-If you were to enter "Foo" as the portlet class name, then the generated project
-would have the following layout:
+If you entered `Foo` as the portlet class name, for example, generates this
+project layout: 
 
 - `com.mycompany.demo.bean.portlet` &rarr; Arbitrary project name.
     - `src/main/java`
-        - `com.mycompany.constants.FooPortletKeys` &rarr; Declares portlet constants.
+        - `com.mycompany.constants.FooPortletKeys` &rarr; Declares portlet 
+        constants.
         - `com.mycompany.portlet.FooPortlet` &rarr; CDI bean portlet class.
-    - `src/main/webapp/WEB-INF/beans.xml` &rarr; Signals CDI to scan the portlet for annotations. 
+    - `src/main/webapp/WEB-INF/beans.xml` &rarr; Signals CDI to scan the 
+    portlet for annotations. 
     - `pom.xml` &rarr; Specifies the project's dependencies and packaging.
 
 When Maven generates the project based on the archetype, it uses the class name
 that you entered for the CDI bean portlet class filename. It also uses it as
 the value of a portlet key constant. 
 
 The generated bean portlet project depends on the Portlet 3.0 API and the "Weld"
-implementation of Contexts and Dependency Injection (CDI). In order to
-facilitate registration of bean portlets, the project also depends on Liferay's
-CDI portable extension. See the project's `pom.xml` for the complete list of
-dependencies.
+implementation of CDI. The project also depends on Liferay's CDI portable
+extension, which facilitates bean portlet registration. The `pom.xml` file lists
```

Comment:
@jhinkey I recommend the word "descriptor" instead of "file" when referring to portlet.xml, liferay-portlet.xml, web.xml, beans.xml, etc.

Reply:
@ngriffin7a - I'm going to leave it as "pom.xml file" as that's how we refer to that file throughout the tooling docs. Thanks.